Source/WebKit/UIProcess/ViewSnapshotStore.cpp

```diff
@@ -93,6 +93,9 @@ void ViewSnapshotStore::recordSnapshot(WebPageProxy& webPageProxy, WebBackForwar
     if (!snapshot)
         return;
 
+#if PLATFORM(MAC)
```

Comment:
At some point we should make this a "behavior flag" not a "which platform" (and also make it control the store size up top), but not in this patch.